src/features/todo-list/todo-list.tsx

```diff
@@ -0,0 +1,32 @@
+import { useTodoStore } from "../../shared/store/useTodoStore";
+import { Todo } from "../todo/todo";
+import styles from "./todo-list.module.scss";
+
+export const TodoList = () => {
+  const { todos } = useTodoStore();
+
+  const activeTodos = todos.filter((todo) => todo.status === "active");
+  const completedTodos = todos.filter((todo) => todo.status === "completed");
```

Comment:
Since there are only two states, I would store the tasks directly in two different arrays, instead of filtering them in each render of the TodoList component.